src/main/java/Duke.java

```diff
@@ -1,10 +1,358 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
+    private static final String SEPARATION_LINE  = "____________________________________________________________";
+    private static final String INDENTATION = "    ";
+    private static final String TEXT_GREETING = "Hello, I'm Tasket\n" + INDENTATION + "What can I do for you?";
+    private static final String TEXT_GOODBYE = "Bye. Hope to see you again soon!";
+    private static final String PATH_FOLDER = "./data";
+    private static final String PATH_FILE = PATH_FOLDER + "/tasks.txt";
+
+    private static String prompt = "";
+    private static ArrayList<Task> tasks = new ArrayList<>();
+    private static File dataFile = new File(PATH_FILE);
+    private static File dataFolder = new File(PATH_FOLDER);
+
+    public static void showSeparationLine() {
+        System.out.println(INDENTATION + SEPARATION_LINE);
+    }
+
+    public static void showGreetingText() {
+        showSeparationLine();
+        System.out.println(INDENTATION + TEXT_GREETING);
+        showSeparationLine();
+    }
+
+    public static void showGoodbyeText() {
+        System.out.println(INDENTATION + TEXT_GOODBYE);
+        showSeparationLine();
+    }
+
+    public static void showErrorText(String msg) {
+        System.out.println(INDENTATION + "OOPS!!! " + msg);
+        showSeparationLine();
+    }
+
+    public static void showLists() {
+        System.out.println(INDENTATION + "Here are the tasks in your list:");
+        for (int i = 1; i <= tasks.size(); i++) {
+            System.out.printf("%s%d. %s\n", INDENTATION, i, tasks.get(i - 1).toString());
+        }
+        showSeparationLine();
+    }
+
+    public static void identifyTaskType(String type, String prompt) throws TasketException {
+        Task task = null;
+        switch (type) {
+        case "todo":
+            task = createToDoTask(prompt);
+            break;
+
+        case "deadline":
+            task = createDeadlineTask(prompt);
+            break;
+
+        case "event":
+            task = createEventTask(prompt);
+            break;
+        }
+
+        FileWriter fw = null;
+
+        try {
+            fw = new FileWriter(dataFile, true);
+            fw.write(task.exportSaveFormat() + System.lineSeparator());
+            tasks.add(task);
+
+            System.out.println(INDENTATION + "Got it, I've added this task:" + System.lineSeparator()
+                    + INDENTATION + task.toString());
+            System.out.printf(INDENTATION + "Now you have %d tasks in the list\n", tasks.size());
+            showSeparationLine();
+        } catch (IOException e) {
+            throw new TasketException("IO Error");
+        } finally {
+            try {
+                if (fw != null) {
+                    fw.close();
+                }
+            } catch (IOException e) {
+                throw new TasketException("IO Error");
+            }
+        }
```

Comment:
Use Java's try-with-resources to eliminate the extra try-catch in the finally block.